### DISCUSSION TITLE
Embedding sections from other Outline articles in current article (i.e. Self-Embedding!)

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#4297] [Ideas]
I commonly am not sure whether I should write this bit of information in this article or create a new article about this topic and then reference this article in this article I am writing. 

One solution is just to create that new article and then insert a link to that page within the article I am currently writing. This, however, is annoying--having to click to another article. 

A better solution: Writing that additional article but them embedding the section (paragraphs) that explain what you need to explain in the article you're in. 

Notion has added something like this. 

This is also a big part of the features that make LogSeq appealing (assuming Roam and Obsidian have this as well).

Food for thought. For now I'll keep just linking to these sections as best I can but this would be awesome.

**Comment by Limezy** [comment]
Yep, the common name for that is transclusion and has become popular with the rise of the so called "PKM" softwares. It can be editable or not. That would be nice to have indeed !
